src/components/UserComboBox.tsx

```diff
@@ -41,26 +63,32 @@ export const UserComboBox = React.forwardRef<HTMLDivElement, UserComboBoxProps>(
                 visible={completionVisible}
                 disabled={disabled}
                 error={error}
-                placement="top-start"
+                placement={placement}
+                offset={offset}
                 items={suggestions.data}
+                onClickOutside={onClickOutside}
                 onChange={onChange}
-                renderTrigger={(props) => (
-                    <Button
-                        ref={props.ref}
-                        text={props.text}
-                        disabled={props.disabled}
-                        onClick={props.onClick}
-                        iconLeft={
-                            value ? (
-                                <UserPic
-                                    src={value?.user?.image}
-                                    email={value?.user?.email || value?.ghost?.email}
-                                    size={16}
-                                />
-                            ) : undefined
-                        }
-                    />
-                )}
+                renderTrigger={(props) =>
```

Comment:
may be in this case use nullish operator ex. `renderTrigger ?? (props) => React.ReactNode`

Reply:
Impossible here. 2 variants based on value.